collector/package.json

```diff
@@ -15,15 +15,13 @@
     "lint": "yarn prettier --ignore-path ../.prettierignore --write ./processSingleFile ./processLink ./utils index.js"
   },
   "dependencies": {
-    "@googleapis/youtube": "^9.0.0",
```

Comment:
unused since we do YT parsing different now

Reply:
https://github.com/Mintplex-Labs/anything-llm/commit/719521c307d81c430d7d90872768d6de556c38d7